modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -96,6 +96,14 @@ To create an API asset:
 . (Optional) Select *Advanced* and edit the advanced settings: *GroupId*, *AssetId*, *Version*, and *API version*. Exchange generates the group ID, asset ID, and version for you, and you can change these values here. You can change an API asset's version (asset version) and API version separately. The advanced settings are most often used to change the asset version.
 . Select *Publish*.
 
+To create a zip file containing all the files in a folder, with the main file in the root directory of the zip file, use a command like this, replacing `myfolder` with the name of your folder and `name.zip` with the name for the new zip file.
```

Comment:
Per style ref., ZIP file
```suggestion
To create a ZIP file containing all the files in a folder, with the main file in the root directory of the ZIP file, use a command like this, replacing `myfolder` with the name of your folder and `name.zip` with the name for the new ZIP file:
```
Can you clarify ", with the main file in the root directory of the ZIP file,"? Do you mean that the ZIP file that you create contains all files in the folder but NOT the main file?

Reply:
You're right, that needs clarification. Rewrote and corrected "ZIP file".